styles/styles.css

```diff
@@ -1094,7 +1091,7 @@ main .block.centered > div {
   }
 
   /* section - narrow */
-  main .section.narrow {
+  main .section.narrow > .section-container {
```

Comment:
Why this changes, this will impact the layout

Reply:
> Why this changes, this will impact the layout

The variants such as `narrow, narrorwer` etc, should apply to `section-container`. The [context is here](https://cq-dev.slack.com/archives/C058KBHLBGR/p1699440902822529). I made a few adjustments to eliminate the impact on layout. Please take a look.